### PR TITLE
[fix-8929][DAO]Description Failed to upload the file because the full name was too long

### PR DESCRIPTION
**Pull request by Hou-Shuaishuai**

### CODE REVIEW

File: dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql [hunk 1]
```diff
@@ -757,7 +757,7 @@ CREATE TABLE `t_ds_resources` (
   `create_time` datetime DEFAULT NULL COMMENT 'create time',
   `update_time` datetime DEFAULT NULL COMMENT 'update time',
   `pid` int(11) DEFAULT NULL,
-  `full_name` varchar(64) DEFAULT NULL,
+  `full_name` varchar(128) DEFAULT NULL,
```
Comment (SbloodyS) [inline dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:760]: Please also add it to postgres and upgrade sql.

Comment (Hou-Shuaishuai) [reply dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:760]: Has been updated

Comment (SbloodyS) [reply dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:760]: You need to create a folder ```2.0.6_schema``` like ```dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.4_schema```. And put ```ALTER TABLE t_ds_resources MODIFY COLUMN NAME VARCHAR(128);``` in ```mysq/dolphinscheduler_ddl.sql```. Same as ```postgresql```.

Comment (Hou-Shuaishuai) [reply dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:760]: It has been modified, please review it again

File: dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.6_schema/postgresql/dolphinscheduler_ddl.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+-- uc_dolphin_T_t_ds_resources_R_full_name
+delimiter d//
```
Comment (SbloodyS) [inline dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.6_schema/postgresql/dolphinscheduler_ddl.sql:18]: Please keep the same style as 
https://github.com/apache/dolphinscheduler/blob/b9448c3f6fc21c9e3030038e76f0f4cf0be768a9/dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.2_schema/postgresql/dolphinscheduler_ddl.sql#L17-L41

For continuous maintenance by others.

Comment (Hou-Shuaishuai) [reply dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.6_schema/postgresql/dolphinscheduler_ddl.sql:18]: please review it again

**Review by SbloodyS — APPROVED** [review]
LGTM, thanks.

Please take a look at it. @zhongjiajie @caishunfeng

**Comment by caishunfeng** [comment]
I had approve to run CI test.

**Comment by SbloodyS** [comment]
@Hou-Shuaishuai Hi, It seems there are some errors in ci. Can you merge the latest dev branch to this pr? And we will rerun it.

**Comment by Hou-Shuaishuai** [comment]
@SbloodyS Have submitted

**Review by caishunfeng — APPROVED** [review]
LGTM

**Comment by caishunfeng** [comment]
> _ No description provided. _

Hi @Hou-Shuaishuai Thanks for your contribution, and next time please add some detail description of this pr.
BTW, this pr close #8929

**Pull request merged**